kiwi/storage/subformat/vagrant_virtualbox.py

```diff
@@ -0,0 +1,432 @@
+# Copyright (c) 2019 SUSE Linux GmbH.  All rights reserved.
+#
+# This file is part of kiwi.
+#
+# kiwi is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# kiwi is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with kiwi.  If not, see <http://www.gnu.org/licenses/>
+#
+
+import os
+
+# project
+from kiwi.storage.subformat.vagrant_base import DiskFormatVagrantBase
+from kiwi.storage.subformat.vmdk import DiskFormatVmdk
+from kiwi.command import Command
+
+
+# Template from which the vagrant boxes' ovf file is created
+#
+# This file has been extracted from a vagrant box that was built with
+# packer. The variable parts are marked via python format strings and were taken
+# from a script provided by Neal Gompa.
+#
+# The following parameters must be passed to this 'template':
+# - vm_name: the name of this VM for virtualbox
+# - disk_image_capacity: Size of the virtual disk image in GB
+# - vm_description: a description of this VM
+# - cimos_id: numeric ID of the operating system in the virtual box (see
+#             :data:`_CIMOS_ID`)
+_OVF_TEMPLATE = """<?xml version="1.0"?>
+<Envelope ovf:version="1.0" xml:lang="en-US" xmlns="http://schemas.dmtf.org/ovf/envelope/1" xmlns:ovf="http://schemas.dmtf.org/ovf/envelope/1" xmlns:rasd="http://schemas.dmtf.org/wbem/wscim/1/cim-schema/2/CIM_ResourceAllocationSettingData" xmlns:vssd="http://schemas.dmtf.org/wbem/wscim/1/cim-schema/2/CIM_VirtualSystemSettingData" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:vbox="http://www.virtualbox.org/ovf/machine">
+  <References>
```

Comment:
Uff, this should really be an extra file, not in place please :)

Reply:
Will do that.